x/msgfee/decorator_test.go

```diff
@@ -0,0 +1,70 @@
+package msgfee
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/coin"
+	"github.com/iov-one/weave/store"
+	"github.com/iov-one/weave/x"
+)
+
+func TestFeeDecorator(t *testing.T) {
```

Comment:
Test looks fine, but what about a table-test here to cover a few cases...

not just valid fee present, but also:
- no fee present (with and without RequiredFee from the handler)
- fee present (with no RequiredFee from handler, with RequiredFee of same token, with RequiredFee of different ticker)

I think that is 5 cases, and you cover 1 here. Some of there are nice edge-cases to ensure working properly (adding with 0 or other currency)

Reply:
Adressed in https://github.com/iov-one/weave/pull/376/commits/a274bf5ce69e3705aaa72a3d3c5423f1a701d84b